package.json

```diff
@@ -27,7 +27,7 @@
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "graphql:codegen": "graphql-codegen --config graphql-codegen.yml",
-    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/varbitrum/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false",
+    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/d8964a5751e8d2077c2db1502c2b429f5c6b50a4/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false",
```

Comment:
Would it make sense to use tagged versions instead? 

<img width="769" alt="image" src="https://github.com/user-attachments/assets/79a23aa5-f9f6-4ba2-a4f3-30cb5c3da0c7">

Reply:
Yes, once they do tag it

Reply:
We are on the bleeding edge 🙃

If they deployed to prod, wouldn't this be tagged already?

Reply:
It has been deployed to prod. However, that version had the issue with open api, which has not been released yet.
And we can't wait for them to deploy this to prod before cutting a version to use on CoW Swap during development.
Anyway, this time I published the RC.0 version and I have this PR pending (https://github.com/cowprotocol/cow-sdk/pull/217) until the version is published to prod.

Reply:
okok 👍